.gitignore

```diff
@@ -170,6 +170,7 @@ concierge.xcodeproj
 tmpimg/
 local
 .eslintrc
+src/
```

Comment:
Reason?

Reply:
running pip install outside of a docker container, it saves gym as a local dep.

Reply:
hmm? not quite sure i understand that connection.

Reply:
I guess the way I think about it is... If there is a potential for it to be committed through normal interaction, and we do not want it to be part of the repo it should be ignored. I am good either way. It was just something I noticed when working on the repo. Thoughts?

I can also give more detail also.

Reply:
I'm still confused. Where is the "src" directory that this is going to ignore?

Reply:
In the root of the project. `./src/`. A sibling to notebooks and games.

Reply:
I am going to push a change that removes the src from the git ignore. We can continue this outside the scope of this PR. I would rather get the docs into master.